projects/plugins/jetpack/extensions/blocks/blogroll/components/blogroll-appender/index.js

```diff
@@ -14,10 +14,11 @@ export default function BlogrollAppender( { subscriptions, clientId } ) {
 	const [ popoverAnchor, setPopoverAnchor ] = useState();
 	const [ searchInput, setSearchInput ] = useState( '' );
 	const { insertBlock } = dispatch( 'core/block-editor' );
-	const { siteDetails } = useGetSiteDetails( {
+	const { siteDetails, isLoading } = useGetSiteDetails( {
 		siteURL: searchInput,
 		subscriptions,
 		enabled: searchInput,
+		enableSiteSearch: true,
```

Comment:
We can get rid of `enableSiteSearch` since we were only using it to temporarily disable the get site details API request.